src/PlaneWaveBasis.jl

```diff
@@ -422,6 +431,34 @@ function weighted_ksum(basis::PlaneWaveBasis, array)
     mpi_sum(res, basis.comm_kpts)
 end
 
+"""
+Utilities to get information about the irreducible k-point mesh (in case of duplication)
+Useful for I/O, where k-point information should not be duplicated
+"""
+function irreducible_kcoords_global(basis::PlaneWaveBasis)
+    # Assume that duplicated k-points are appended at the end of the kcoords array
+    basis.kcoords_global[1:basis.n_irreducible_kpoints]
+end
+
+function irreducible_kweights_global(basis::PlaneWaveBasis{T}) where {T}
+    @inline function same_kpoint(i_irr, i_dupl)
```

Comment:
You need the `@inline` ? Is the compiler not smart enough to do this ?
Also anyway not performance crtical ...

Reply:
Probably overkill, I agree